### ISSUE TITLE
Problem getting authentication token

### ISSUE THREAD

**Issue by pfzlsk** [ukdtom/ClaimIt#29]
![image](https://user-images.githubusercontent.com/97282473/148518958-125704ae-bc27-4676-b824-57f18544fa2a.png)

**Comment by pfzlsk** [comment]
Both my PMS and mt web interface are on the same PC. Any suggestions?

**Comment by ukdtom** [comment]
What OS and version?

**Comment by pfzlsk** [comment]
![image](https://user-images.githubusercontent.com/97282473/148586430-2e65c35f-ab08-4960-af11-0074833f92f8.png)

**Comment by pfzlsk** [comment]
My Plex installation was previously working and stable, but then in early December I started to lose connectivity with the server frequently. I noticed the failed install message and tried to reinstall but then the Claim Server button appeared. I have totally removed PMS  using Revo uninstaller to purge all registry entries and delete all remaining files, but with no success. I even dropped back to a previous PMS release 1.23.6.4881

**Comment by ukdtom** [comment]
It's a Windows issue, where it can't connect to plex.tv using the newer encryption req. by Plex

Make sure, that you are running the latest .Net4 version avail for a start
https://dotnet.microsoft.com/en-us/download/dotnet-framework/thank-you/net48-web-installer

And if not working, then you need to troubleshot your windows OS, which is out of scope for ClaimIT

**Comment by pfzlsk** [comment]
I looked at the registry entry and I am running the .net 4.8 framework I have also tried running the repair tool from Microsoft but that didn't work, and I've kind of exhausted my knowledge set at this point. I appreciate your help and just knowing the general location of the problem is a tremendous help. I know it's beyond the scope of ClaimIT but do you have any suggestions on where I should look next to fix this in the OS? Do you think that a brute force approach of upgrading top Win 11 might work?

**Comment by ukdtom** [comment]
Sadly no idea, since I'm a Linux dude, so sadly can't help here....

But when said...

Can you using a browser on the box reach https://plex.tv ?

**Comment by pfzlsk** [comment]
Pops open immediately

**Comment by ukdtom** [comment]
DArn, then it actually might be my problem ;)

ClaimIt was compiled with .Net4.5

Above .Net version did support TLS1.2, but at that time, it was not yet the standard, and as such, TLS1.1 was used as the default.

Plex no longer supports TLS 1.1 ;)

Stand by here

**Comment by ukdtom** [comment]
[ClaimIt.zip](https://github.com/ukdtom/ClaimIt/files/7832111/ClaimIt.zip)

Does above work?

**Comment by pfzlsk** [comment]
When I ran it, it flashed some gray text with Password OK in green, then cleared the message box, then after a few seconds went to Not Responding. I did not get an error message.

**Comment by pfzlsk** [comment]
It did generate a log request to Plex however, as I received a new device login email from them

**Comment by ukdtom** [comment]
Hmm, and okay....Will look more into this, but sadly 3.20 at night here, so ;)

**Comment by pfzlsk** [comment]
I may not have let it run long enough. I tried again and got the following, which looks like progress....
![image](https://user-images.githubusercontent.com/97282473/148627971-2555c6a9-46da-437f-8260-59cf322dd4e0.png)

**Comment by ukdtom** [comment]
That is much better, and look here:

https://github.com/ukdtom/ClaimIt/wiki/Errors-and-Troubleshooting#500

**Comment by pfzlsk** [comment]
So I checked the wiki and I did not edit preferences as this is a clean install, and the ping test worked. My DNS is provided by my Ubiquiti gateway, so I will attempt a re-install.

**Comment by ukdtom** [comment]
Updated my comment above !

**Comment by pfzlsk** [comment]
I followed the "Why am I locked out" directions with the same results. I only had one of the reg entries of the 4 it said to look for. I also ran ClaimIT after getting the unclaimed message and again got the 500 error. 

Earlier, I tried using the installer's repair function and curiously it stopped to ask for the pms.exe file which did not exist in the location (at least outside of the installer file), but at the point it stopped the installer window referenced the C++ Redistributable (2015-2019), which I confirmed is installed. So this may point to deeper problems with the OS. So I'm back at Square 1. 

I really appreciate all of your effort
